presidio-analyzer/Dockerfile.windows

```diff
@@ -1,6 +1,6 @@
 FROM python:3.9-windowsservercore
 
-ARG NLP_CONF_FILE=conf/default.yaml
+ARG NLP_CONF_FILE=presidio_analyzer/conf/transformers.yaml
```

Comment:
default.yaml

Reply:
check out docker-compose-transformers.yaml too

Reply:
There it is supposed to be tramsformers.yaml, doesn't it?

Reply:
Fixed it docker-compose-transformers